2019Labs/CustomSecurityContent/documentation/lab4_ansible.adoc

```diff
@@ -311,52 +354,64 @@ To use our Playbook on your machine (on a local host), replace the placehoder st
 
 Notice that the timeout value supplied by variable `var_accounts_tmout` has been set to a specific value (1800 seconds) during the build process, and the variable has been added to the `vars` section of the Playbook.
 
-Notice also that the Playbook has tags in the `tags` section which were added based on metadata in `rule.yml`. At the beginning, it contains the CCE (Common Configuration Enumeration) identifier.
+Notice also that the Playbook has tags in the `tags` section which were added based on metadata in `rule.yml`.
+At the beginning, it contains the CCE (Common Configuration Enumeration) identifier.
 
 Finally, the `tasks:` section contains our Ansible task which we created.
 
+In case there is a typo in the YAML file, edit the source again and rebuild.
+After that, you will need to again replace the placeholder string `'@@HOSTS@@'`.
+
+----
+[... lab4_ansible]$ nano linux_os/guide/system/accounts/accounts-session/accounts_tmout/ansible/shared.yml
+[... lab4_ansible]$ ./build_product rhel8
+[... lab4_ansible]$ nano build/rhel8/playbooks/ospp/accounts_tmout.yml
+----
 
 Run the following command to run the Playbook:
 
 ----
-$ ansible-playbook build/rhel8/playbooks/ospp/accounts_tmout.yml
+[... lab4_ansible]$ ansible-playbook build/rhel8/playbooks/ospp/accounts_tmout.yml
 ----
 
 Check if it has any effect:
 
 ----
-$ gedit /etc/profile
+[... lab4_ansible]$ cat /etc/profile
 ----
 
 Notice that `TMOUT=1800` is at the end of the file!
 
 The biggest advantage of using Ansible tasks in ComplianceAsCode is that it gets integrated with the SCAP content and also the HTML report and in the HTML guide as well.
 
+Switch to the console view, open the terminal.
 Run the following command to open the HTML guide for OSPP profile for Red Hat Enterprise Linux 8 in your Firefox web browser:
 
 ----
-$ firefox build/guides/ssg-rhel8-guide-ospp.html
+[... ~]$ firefox /home/lab-user/labs/lab4_ansible/build/guides/ssg-rhel8-guide-ospp.html
```

Comment:
Please consider adding something like "use this terminal command or navigate to the OSPP guide in the same way as you did in previous exercises".

Reply:
did so, thanks for the suggestion!